cpp/tensorrt_llm/batch_manager/trtGptModelInflightBatching.h

```diff
@@ -489,6 +499,8 @@ class TrtGptModelInflightBatching : public TrtGptModel
     std::optional<SizeType32> mMaxNumTokensRuntime;
     // Controls if generation logits should be gathered, so that returnGenerationLogits can be requested.
     bool mGatherGenerationLogits{false};
+    // Whether the context is chunked
```

Comment:
the description can made more clearer like `offloading and prefetching the prompt tuning table (only effective in chunked prefill mode)`